services/speech_to_text/v1.js

```diff
@@ -63,14 +63,20 @@ module.exports = function (RED) {
       }
 
       if (!config.lang) {
-        var message = 'Missing audio language configuration, unable to process speech.';
-        node.error(message, msg)
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'message_err_lang' is not in camel case.](https://www.codacy.com/app/rezgui-y/node-red-node-watson/file/2391621463/issues/source?bid=3225115&fileBranchId=3229003#l66)